internal/test/framework/windowsvm.go

```diff
@@ -104,6 +107,43 @@ func newWindowsVM(imageID, instanceType string, credentials *types.Credentials,
 	return w, nil
 }
 
+func (w *testWindowsVM) CopyFileTo(filePath, remoteDir string) error {
```

Comment:
Is this the same as windowsVM.CopyFileTo?
If so, we shouldn't be duplicating code. You should just call the `Windows` implementation of CopyFileTo.